### PR TITLE
PAQ change log entry for infinite loop with pageSize (PAM-34761)

### PR DESCRIPTION
**Pull request by kailol-sag**

### CODE REVIEW

**Review by bph-sag — APPROVED** [review]

File: content/change-logs/analytics/apama-in-c8y-20240514-Queries-with-pageSize-set-to-1-no-longer-loop-infinitely.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date:
+title: Queries with pageSize set to 1 no longer loop infinitely
```
Comment (Rob-J-SAG) [inline content/change-logs/analytics/apama-in-c8y-20240514-Queries-with-pageSize-set-to-1-no-longer-loop-infinitely.md:3]: Think we need to qualify queries as "REST queries" or "API queries". Apply the same to the description text too.

Comment (bph-sag) [reply content/change-logs/analytics/apama-in-c8y-20240514-Queries-with-pageSize-set-to-1-no-longer-loop-infinitely.md:3]: If there's any previous precedence for terminology here, go with that, I'd personally go with "REST queries".

Comment (kailol-sag) [reply content/change-logs/analytics/apama-in-c8y-20240514-Queries-with-pageSize-set-to-1-no-longer-loop-infinitely.md]: I change it to "REST queries".

**Review by Rob-J-SAG — APPROVED** [review]

**Comment by skom-sag** [comment]
v25.167.0 is deployed to eu-latest on 11 June 2024

**Pull request merged**